Client/multiplayer_sa/CMultiplayerSA_1.3.cpp

```diff
@@ -1435,7 +1435,7 @@ void                       CVehicleModelInterface_SetClump()
 {
     // Loop through all vehicles and find the vehicle id that this interface belongs to
     CModelInfo* pModelInfo = NULL;
-    for (int i = 400; i < 612; i++)
+    for (int i = 400; i < 80000000; i++)
```

Comment:
Why do you need that?

Reply:
It adds more than one parts. Here you specified only the GTA vehicle ids for (int i = 400; i < 612; i++), but it was not checking when there were more ids, so I increased the number. This is a temporary solution. I could not connect the deatmach file to the multiplayer:sa file to capture the parent id. I did something like this for him and it worked.

Reply:
It is extremely inefficient and can lead to freezes when spawning vehicles.

Reply:
How else can I prevent it? I already thought of this I put it as a temporary solution and it worked

Reply:
How else can I prevent it? I already thought of this I put it as a temporary solution and it worked

Reply:
@tederis 
I updated it again, can you check it? I tested it and it worksworks